holoviews/core/spaces.py

```diff
@@ -441,6 +455,8 @@ class Callable(param.Parameterized):
     def __init__(self, callable, **params):
         super(Callable, self).__init__(callable=callable, **params)
         self._memoized = {}
+        self._overlay = False
```

Comment:
Maybe ``self._is_overlay`` to make it clearer it is a predicate as opposed to a command?